packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -516,12 +516,20 @@ export const createAzureCommunicationCallAdapter = async ({
   locator
 }: AzureCommunicationCallAdapterArgs): Promise<CallAdapter> => {
   const callClient = createStatefulCallClient({ userId });
-  const deviceManager = (await callClient.getDeviceManager()) as StatefulDeviceManager;
   const callAgent = await callClient.createCallAgent(credential, { displayName });
-  const adapter = new AzureCommunicationCallAdapter(callClient, locator, callAgent, deviceManager);
+  const adapter = createAzureCommunicationCallAdapterFromClient(callClient, callAgent, locator);
   return adapter;
 };
 
+export const createAzureCommunicationCallAdapterFromClient = async (
```

Comment:
(non PR blocking comment) Do we need/want storybook documentation for this?

Reply:
Yea, we should have a work item for this, but not necessary for GA or azure review